### DISCUSSION TITLE
This was better than the OpenAI app

### DISCUSSION THREAD

**Discussion by lduperval** [lencx/ChatGPT#1414] [Ideas]
Hi,

The official OpenAI app is nothing more than a simple frontend for the website. It has no other ability… except maybe the use of Alt-Spavcebar to bring up the app. But even that conflicts with PowerToys…

Simply making the 1.1.0 interface usable with the new version of the website would make this app better.

Some ideas that make it better:

- make the "Save entire conversation" as markdown or PDF work again
- The ability to disable/enable the buttons for this feature
- Search conversations (including archived ones)
- Grouping of conversations

**Comment by ellielo** [comment]
Sure, the official OpenAI app often comes across as bare-bones rather than packed with features. Lately I've turned to drafting inside it first, then shifting over to Clever AI Humaniser [https://cleverhumanizer.ai/](https://cleverhumanizer.ai/) to smooth things out - somehow it adds that real person touch without dragging. Free of charge, zips through edits, keeps every version you reshaped, yet now and again I’ll still nudge the wording just a little. The flow feels off otherwise.
Your suggestions sound useful - saving talks as markdown or PDF might help a lot. Searching past chats could change how people use the app every day. Grouping messages together adds another layer of ease. Small tweaks matter too, like turning options on or off when needed. A cleaner look would go a long way. At present, the site handles most heavy tasks. Moving those tools into the mobile version would shift things noticeably. Writing gets smoother. Research feels quicker. Staying organized becomes simpler. Seeing OpenAI include even one or two of these changes sometime soon would mean something.
